cmake/lib-config.cmake.in

```diff
@@ -0,0 +1,4 @@
+@PACKAGE_INIT@
```

Comment:
What is this for?

Reply:
Seems like it customizes some installation options. Looks to be the same as we use in rdf4dpp.

Reply:
mmh, never noticed it before